### PR TITLE
[DOCS] Tweak addObserver function signature examples

### PR DESCRIPTION
**Pull request by knownasilya**
The semicolons would have been invalid JS. I also converted to ES2015 while I was here, let me know if that's fine.

### CODE REVIEW

File: packages/ember-runtime/lib/mixins/observable.js [hunk 1]
```diff
@@ -352,7 +352,7 @@ export default Mixin.create({
     you do not pass a `context` parameter:
 
     ```javascript
-    fooDidChange: function(sender, key, value, rev) { };
```
Comment (locks) [inline packages/ember-runtime/lib/mixins/observable.js]: Can you wrap the examples in an object? I'm guessing this minimal code sample might be confusing.

**Review by locks — CHANGES_REQUESTED** [review]
Thanks for spotting this.
Can you improve it a bit more since you're in there?

File: packages/ember-runtime/lib/mixins/observable.js (hunk 1 above)
Comment (knownasilya) [inline packages/ember-runtime/lib/mixins/observable.js]: should i add an example like

```js
export default Ember.Component.extend({
  init() {
    this._super(...arguments);
    this.addObserver('someProperty', this, 'aFunc');
  },

  aFunc(sender, key, value, rev) {
    // your code
  }
});
```

Comment (rwjblue) [inline packages/ember-runtime/lib/mixins/observable.js]: @knownasilya - Yes, that looks much better to me...

**Review by locks — APPROVED** [review]

**Comment by locks** [comment]
Thank you very much!

**Pull request merged**